app.css

```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
```

Comment:
there is no need of border-radius property to be used of !!

Reply:
If I don't add this then my picture wouldn't get rounder.